src/examples/src/simple-component/App/template.html

```diff
@@ -1,9 +1,10 @@
 <ol>
   <!--
-    We are providing each todo-item with the todo object
-    it's representing, so that its content can be dynamic.
-    We also need to provide each component with a "key",
-    which is explained in the guide section on v-for.
+    Forniamo a ogni todo-item l'oggetto todo
+    che rappresenta, in modo che il suo contenuto
+    possa essere dinamico. Dobbiamo fornire a ogni
```

Comment:
C'è una riga in più:
4: (... ) possa essere dinamico 
5: Dobbiamo fornire anche una "chiave" per ogni componente,